### ISSUE TITLE
Duplicated assets in pubspec.yaml

### ISSUE THREAD

**Issue by Ricardo-MT** [rutvik110/asset_manager_cli#23]
Description:
When running asset_manager add on a project that already has 'assets:' defined in the pubspec.yaml file, this package reads all subdirectories in the root/assets/ directory and appends a second property 'assets:' with all found paths after the 'flutter:' property in pubspec.yaml file, instead of replacing the existing one.

pubspec.yaml file before running the package:
```yaml
flutter:

  assets:
    - assets/audio/
    - assets/images/
    - assets/images/1_Interiors/
    - assets/images/1_Interiors/32x32/
```

pubspec.yaml file after running the package:
```yaml
flutter:

  assets:
    - assets/audio/
    - assets/images/
    - assets/images/1_Interiors/
    - assets/images/1_Interiors/32x32/

  assets:
    - assets/audio/
    - assets/images/
    - assets/images/1_Interiors/
    - assets/images/1_Interiors/32x32/
```

Expected behaviour:
Before adding the generated 'assets:' property with all found subdirectories, the package should remove (if exists) the current 'assets:' property.

**Comment by rutvik110** [comment]
Fixed in latest release `1.0.1`.

**Comment by Ricardo-MT** [comment]
I can confirm, this issue is fixed in latest release 1.0.1.

However, if the section "assets:" already exists running the command adds a new line at the end of the section. If I run the command 5 times it adds 5 new lines.
